src/redismodule.rs

```diff
@@ -155,6 +168,19 @@ impl Drop for RedisString {
     }
 }
 
+impl Display for RedisString {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "{}", self.into_string_lossy())
+    }
+}
+
+impl Borrow<str> for RedisString {
+    fn borrow(&self) -> &str {
+        // RedisString might not be UTF-8 safe
+        self.try_as_str().unwrap_or_default()
```

Comment:
The default may be a bit weird since it'll be empty. It might be better to predefine a placeholder string such as `"<Invalid UTF-8 data>"` and return that (via `impl Default`?).

Reply:
```
        self.try_as_str().unwrap_or_else("<Invalid UTF-8 data>")
```